.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 .DS_Store
 .idea
 storybook-static
+.rollup.cache
```

Comment:
With the new changes to the build, there was a subfolder for .rollup.cache which we don't want as part of the repo so I've ignored it. It's a build artifact, the whole folder.